.pipelines/OneBranch.Official.yml

```diff
@@ -37,8 +37,16 @@ extends:
       enabled: false
     
     globalSdl:
+      isNativeCode: true
+      asyncSdl:
+        enabled: true
       tsa:
-        enabled: false
+        enabled: true
```

Comment:
Does this work?

Reply:
We'll kick off an official build and find out. At least the work is likely complete from our end.